tests/test_diffusers.py

```diff
@@ -135,32 +131,30 @@
     THROUGHPUT_BASELINE_AUTOCAST = 0.394
     TEXTUAL_INVERSION_THROUGHPUT = 131.7606336456344
     TEXTUAL_INVERSION_RUNTIME = 1.542460777796805
-    CONTROLNET_THROUGHPUT = 120.123522340414
-    CONTROLNET_RUNTIME = 1.8647471838630736
-    INPAINT_THROUGHPUT_BASELINE_BF16 = 4.584
-    INPAINT_XL_THROUGHPUT_BASELINE_BF16 = 1.151
-    TEXT_TO_VIDEO_SYNTHESIS_BF16_BASELINE = 70
-    DETERMINISTIC_IMAGE_GENERATION_THROUGHPUT = 0.946
-    THROUGHPUT_UNCONDITIONAL_IMAGE_BASELINE_BF16 = 0.15186785472532677
-    DEPTH2IMG_GENERATION_LATENCY_BASELINE_BF16 = 36.06376791000366
```

Comment:
Not opposed to it, but why removing this one or `DETERMINISTIC_IMAGE_GENERATION_THROUGHPUT` ?

Reply:
We removed some outdated and less main-stream samples/tests, this is one of those.   Trying to minimize number of slow tests and keep basic pipeline slow tests.

Reply:
Sounds good!